CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## Unreleased
+* d/tfe_organization: Add `default_project_id` field.
```

Comment:
Update the changelog lines to follow format with the PR numbers, like : 
```
d/tfe_organization: Add `default_project_id` field. ([#704](https://github.com/hashicorp/terraform-provider-tfe/pull/704))
```

Reply:
Done!